src/test/java/com/google/api/codegen/testdata/python_main_no_path_templates.baseline

```diff
@@ -161,16 +161,16 @@ class NoTemplatesAPIServiceClient(object):
         whitespace.
 
         Example:
-          >>> import example
-          >>> client = example.NoTemplatesAPIServiceClient()
-          >>> client.increment()
+            >>> import example
+            >>> client = example.NoTemplatesAPIServiceClient()
+            >>> client.increment()
 
         Args:
-          options (:class:`google.gax.CallOptions`): Overrides the default
-            settings for this call, e.g, timeout, retries etc.
+            options (~.google.gax.CallOptions): Overrides the default
```

Comment:
Remove the leading dot if going with the full path. So, `~google.gax.CallOptions`. Sorry, I should have been more clear on this.